Modules/Microsoft365DSC/DSCResources/MSFT_TeamsUpgradeConfiguration/MSFT_TeamsUpgradeConfiguration.schema.mof

```diff
@@ -4,6 +4,9 @@ class MSFT_TeamsUpgradeConfiguration : OMI_BaseResource
     [Key, Description("Specifies the resource is a single instance, the value must be 'Yes'"),ValueMap{"Yes"},Values{"Yes"}] String IsSingleInstance;
     [Write, Description("The DownloadTeams property allows admins to control whether the Skype for Business client should automatically download Teams in the background. This Boolean setting is only honored on Windows clients, and only for certain values of the user's TeamsUpgradePolicy. If NotifySfbUser=true or if Mode=TeamsOnly in TeamsUpgradePolicy, this setting is honored. Otherwise it is ignored.")] Boolean DownloadTeams;
     [Write, Description("The SfBMeetingJoinUx property allows admins to specify which app is used to join Skype for Business meetings, even after the user has been upgraded to Teams. Allowed values are: 'SkypeMeetingsApp' and 'NativeLimitedClient'. 'NativeLimitedClient' means the existing Skype for Business rich client will be used, but since the user is upgraded, only meeting functionality is available. Calling and Messaging are done via Teams. 'SkypeMeetingsApp' means use the web-downloadable app. This setting can be useful for organizations that have upgraded to Teams and no longer want to install Skype for Business on their users' computers."),ValueMap{"SkypeMeetingsApp","NativeLimitedClient"}, Values{"SkypeMeetingsApp","NativeLimitedClient"}] String SfBMeetingJoinUx;
-    [Required, Description("Credentials of the Teams Admin"), EmbeddedInstance("MSFT_Credential")] string Credential;
+    [Write, Description("Credentials of the Teams Admin"), EmbeddedInstance("MSFT_Credential")] string Credential;
```

Comment:
Please check the various Readme files for consistency. Some say "Teams Global Admin", this one says "Teams Admin".

I think it makes most sense to use "Teams Admin".

Reply:
Done. Changed "Teams Global Admin" to "Teams Admin" across the board.